centraldashboard/rockcraft.yaml

```diff
@@ -1,33 +1,40 @@
+# Dockerfile https://github.com/kubeflow/kubeflow/blob/v1.7-branch/components/centraldashboard/Dockerfile
 name: kubeflow-central-dashboard
 summary: Kubeflow Landing Page
 description: |
   This component serves as the landing page and central dashboard for Kubeflow deployments.
   It provides a jump-off point to all other facets of the platform.
-version: 1.7.0_22.04_1 # version format: <KF-upstream-version>_<base-version>_<Charmed-KF-version>
+version: "1.7"
```

Comment:
Why is the version 1.7, and getting merged to main, if the goal of the task is to have this in 1.8?

Reply:
@kimwnasptd you are right in that we want to upgrade for 1.8, I must've been focused on updating the rock and making it work that I forgot the version. Will update. 

@misohu I will have your suggestion in mind, thanks!

Reply:
@misohu about the node version, [this commit](https://github.com/canonical/kubeflow-rocks/commit/82e3850f8d6baffc8d96d4bf7fa42905bf4d30f7) introduced the change, but there is no apparent justification. I am setting it to node 14 to keep consistency with upstream.
About the user, unfortunately it is a requirement by the security team and I don't think we can change it from __daemon__ to node, and tbf I don't think there will be an issue.

Reply:
yeah we should always update the node to latest possible for critical vulnerabilities ... yeah the user makes sense if its not doing problems looking good